src/main/kotlin/com/atlassian/performance/tools/awsinfrastructure/api/jira/DataCenterFormula.kt

```diff
@@ -208,6 +209,7 @@ class DataCenterFormula private constructor(
             }
             .mapIndexed { i: Int, instance ->
                 val ssh = Ssh(SshHost(instance.publicIpAddress, "ubuntu", keyPath), connectivityPatience = 5)
+                val isMySql = database is MySqlDatabase
```

Comment:
We've got a better way of doing this now. I'll pick pieces of code from this PR and stitch them into the new hook-based API. Basically we won't even change a byte in the old `DataCenterFormula`, `StandaloneFormula`, yamls, etc.